net/nextdns/Makefile

```diff
@@ -0,0 +1,80 @@
+#
+# Copyright (C) 2012-2019 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=nextdns
+PKG_VERSION:=1.1.1
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_VERSION:=v$(PKG_VERSION)
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_SOURCE_VERSION).tar.xz
+PKG_SOURCE_URL:=https://github.com/nextdns/nextdns.git
+PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_SOURCE_VERSION)
+PKG_MIRROR_HASH:=0f49d56c294cad71f8abc18d8f1997c2c76732427f6e6ce0dac19f59dd32455d
+PKG_BUILD_DIR:=$(BUILD_DIR)/nextdns-$(PKG_SOURCE_VERSION)
+
+PKG_MAINTAINER:=Olivier Poitrey <rs@nextdns.io>
+PKG_LICENSE:=MIT
+PKG_LICENSE_FILES:=LICENSE
+
+PKG_BUILD_DEPENDS:=golang/host
+PKG_BUILD_PARALLEL:=1
+PKG_USE_MIPS16:=0
+
+GO_PKG:=github.com/nextdns/nextdns
+
```

Comment:
Can add:

```make
GO_PKG_LDFLAGS_X:=main.version=$(PKG_VERSION)
```

to set the version string inside your binary.

Reply:
Good point.